CHANGELOG.md

```diff
@@ -1,4 +1,100 @@
-<a name="9.10.0></a>
+<a name="10.0.0"></a>
+
+# [10.0.0](https://github.com/commercetools/ui-kit/compare/v9.10.0...10.0.0) (2019-06-20)
+
+### Breaking Changes
+
+This release introduces **breaking changes** which may entail some **migration steps**. We'll go through them now:
+
+- `hocs`: `withMouseOverState` and `withMouseDownState` have been dropped. Please use CSS properties instead. ([#881](https://github.com/commercetools/ui-kit/pull/881))
```

Comment:
I would maybe add that if people need those, we recommend using hooks. We can also provide a hook snippet that people can copy-paste.

Redux does something similar and I found it very useful.
https://react-redux.js.org/next/api/hooks#hooks-recipes

Reply:
I don't think this is necessary since we already made these HOCs internal to the MC.